src/structures/Guild.js

```diff
@@ -311,6 +312,12 @@ class Guild extends Base {
      */
     this.vanityURLCode = data.vanity_url_code;
 
+    /**
+     * The use count of the vanity URL code of the guild, if any
```

Comment:
should there be a <warn> (or <info>) tag here to say it must have been fetched once?

Reply:
For this, it's only populated on Guild#fetchVanityData but adding that to the @link pushes it over jsdoc eslint rules.

Should I just disable the rule for tha that line?

Reply:
Yes, disabling it is fine then.